docs/rule/no-childless-elements.md

```diff
@@ -0,0 +1,45 @@
+## no-childless-elements
+
+Childless elements
```

Comment:
Remove this line that just says `Childless elements`?

Reply:
Yes @bmish and it also tag home made components, like if I had a `<Sidebar />` 😓 

I suggest to inverse the logic here and rename this rule.

The idea would be to to have a rule named `flag-childless-elements`.
It would only work with a custom config passed, no tag flagged by default.

Then you could have in your `.template-lintrc.js` something like:
```
'no-childless-elements': {
  tags: ['div', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6'],
},
```

Then you'll only have and error if those tags' node do not have children (`!node.children.length`)

What do you think of this implementation ?